WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -53,7 +56,11 @@ struct HubMenu: View {
             ServiceLocator.analytics.track(event: .Blaze.blazeCampaignListEntryPointSelected(source: .menu))
         }
 
-        viewModel.selectedMenuID = menu.id
+        if menu.id == HubMenuViewModel.GoogleAds.id {
+            googleAdsCampaignHandler()
+        } else {
```

Comment:
Suggestion: Instead of adding an if else condition, would using a guard to exit early after tracking `hubMenuOptionTapped` make it more readable?

Reply:
I think both looks the same 🤔 There's not much logic after the `guard` check so I don't see much improvement when using it.